install.rdf

```diff
@@ -26,15 +32,93 @@
+        <em:contributor>John Bieling</em:contributor>
+        <em:contributor>Philippe Lieser</em:contributor>
+        <em:translator>(de)</em:translator>
+        <em:translator>Emil Hesslow (en-US)</em:translator>
+        <em:translator>(fr)</em:translator>
+        <em:translator>Óvári (hu)</em:translator>
+        <em:translator>(ja)</em:translator>
+        <em:translator>(sv-SE)</em:translator>
+      </Description>
+    </em:localized>
+
     <!-- Hungarian -->
     <em:localized>
       <Description>
         <em:locale>hu</em:locale>
         <em:name>Gyorsszöveg (Quicktext)</em:name>
         <em:description>Eszköztár, parancsfájlok és sablonok korlátlan lehetőségekkel, beleértve a [[Címzett=Keresztnév]] változókat, a gyors szövegbeillesztéshez</em:description>
         <em:contributor>R Kent James</em:contributor>
-        <em:contributor>John Bieling</em:contributor>    
+        <em:contributor>John Bieling</em:contributor>
+        <em:contributor>Philippe Lieser</em:contributor>
+        <em:translator>(de)</em:translator>
+        <em:translator>Emil Hesslow (en-US)</em:translator>
+        <em:translator>(fr)</em:translator>
+        <em:translator>Óvári (hu)</em:translator>
+        <em:translator>(ja)</em:translator>
+        <em:translator>(sv-SE)</em:translator>
+      </Description>
+    </em:localized>
+
+    <!-- Japanese -->
+    <em:localized>
+      <Description>
+        <em:locale>ja</em:locale>
+        <em:name>Quicktext</em:name>
+        <em:description>Adds a toolbar with unlimited number of text to quickly insert. It’s also possible to use variables like [[TO=firstname]]. With settings for everything.</em:description>
+        <em:contributor>R Kent James</em:contributor>
+        <em:contributor>John Bieling</em:contributor>
+        <em:contributor>Philippe Lieser</em:contributor>
+        <em:translator>(de)</em:translator>
+        <em:translator>Emil Hesslow (en-US)</em:translator>
+        <em:translator>(fr)</em:translator>
+        <em:translator>Óvári (hu)</em:translator>
+        <em:translator>(ja)</em:translator>
+        <em:translator>(sv-SE)</em:translator>
+      </Description>
+    </em:localized>
+
+    <!-- Swedish -->
+    <em:localized>
+      <Description>
+        <em:locale>sv-SE</em:locale>
+        <em:name>Quicktext</em:name>
+        <em:description>Adds a toolbar with unlimited number of text to quickly insert. It’s also possible to use variables like [[TO=firstname]]. With settings for everything.</em:description>
+        <em:contributor>R Kent James</em:contributor>
+        <em:contributor>John Bieling</em:contributor>
+        <em:contributor>Philippe Lieser</em:contributor>
+        <em:translator>(de)</em:translator>
+        <em:translator>Emil Hesslow (en-US)</em:translator>
+        <em:translator>(fr)</em:translator>
         <em:translator>Óvári (hu)</em:translator>
+        <em:translator>(ja)</em:translator>
+        <em:translator>(sv-SE)</em:translator>
```

Comment:
I do not know any of the missing translator, please remove placeholders

Reply:
Do you you prefer `Anonymous (ja)` like the [Mail Merge add-on](https://addons.thunderbird.net/addon/mail-merge/) as shown in the image below?

![kep](https://user-images.githubusercontent.com/17465872/47588604-711e0900-d9b2-11e8-8ae2-4760a6ebefad.png)

Reply:
I would go for Unknown, what do you think?

Reply:
Done